workshop/exercise-6/README.md

```diff
@@ -163,7 +165,22 @@ default-image: my-nodejs-express
 default-pipeline: default
 images:
 - id: my-nodejs-express
-  image: $IMAGE_REGISTRY_ORG/my-nodejs-express:0.2
+  image: $IMAGE_REGISTRY_ORG/my-nodejs-express:0.3
+```
+
+Edit the new file called `stack.yaml` in `collections/incubator/my-nodejs-express` (you will probably see a different version, that's ok to keep as-is):
```

Comment:
Maybe we should say which fields they should be updating....?

Reply:
good idea... done along with an admonishment not to change the version to match the example.